src/scss/initial/_themes.scss

```diff
@@ -0,0 +1,24 @@
+@use '../config/color' as config;
+@use '../../../node_modules/cascading-color-systems' as ccs;
+
+// OddBird
+// -------
+/// The OddBird theme is based on default settings,
+/// but removes user hue settings.
+/// @name Default Theme
+/// @group init-themes
+[data-ccs-theme='oddbird'] {
+  --ccs-prime--theme: var(--ccs-prime--config);
+  --ccs-accent--theme: var(--ccs-accent--config);
+  --ccs-custom-hue: none;
```

Comment:
It removes the hue setting, but if you switch to another theme, change the hue, and switch back it still keeps the active hue of whatever theme I as just looking at, right? Until you hit reset anyway.

Reply:
that's a bug! fixing…